scripts/extract-formatting-changes.ts

```diff
@@ -0,0 +1,130 @@
+import { exec, execSync } from 'child_process';
+import yargs from 'yargs';
+import * as fs from 'fs';
+import * as path from 'path';
+import * as readline from 'readline';
+
+const getArgs = (latestRevision: string) => {
+  return yargs(process.argv.slice(2))
+    .usage('Usage: $0 -s [rev1] -h [rev2] -b [branch]')
+    .nargs('s', 1)
+    .describe('s', 'Base revision')
+    .default('s', 'dev')
+    .nargs('h', 1)
+    .describe('h', 'Head revision')
+    .default('h', latestRevision)
+    .nargs('b', 1)
+    .default('b', '')
+    .describe('b', 'Branch name').argv;
```

Comment:
I liked the verbose `base` `head` and `branch` alternatives :-(

Reply:
I wasn't getting the expected behavior from the aliases, so I ended up removing them.